api-tests/src/steps/ipv-steps.ts

```diff
@@ -206,7 +208,7 @@ When(
 
 // Variant of the journey start that retries, e.g. to wait for an async F2F request
 When(
-  /I start a new '([\w-]+)' journey( with reprove identity)? and return to a '([\w-]+)' page response$/,
+  /I start new '([\w-]+)' journeys( with reprove identity)? until I get a '([\w-]+)' page response$/,
```

Comment:
I agree the phrase needed an update though I think it would make more sense implying you're attempting to start the journey.
```suggestion
  /I attempt to start a new '([\w-]+)' journey( with reprove identity)? until I get a '([\w-]+)' page response$/,
```